lib/msal-core/src/Configuration.ts

```diff
@@ -38,6 +38,7 @@ export type AuthOptions = {
   redirectUri?: string | (() => string);
   postLogoutRedirectUri?: string | (() => string);
   navigateToLoginRequestUrl?: boolean;
+  responseType? : string;
```

Comment:
I dont think this belongs in the top level config, if it were to be added wouldn't it make sense in the AuthenticationParameters request for a single request ?

Reply:
We need a more streamlined approach to handle idToken only cases but I don't believe we can make it a surface param. I can check with the PMs on what our guidance is here. Also as I stated above,  @anjiuidev use case could be solved without this if we figure out why the delay.